_integration-tests/utils/generator/main.go

```diff
@@ -33,12 +57,51 @@ func main() {
 		return strings.Compare(lhs.Name(), rhs.Name())
 	})
 
+	dc := decorator.NewDecorator(nil)
```

Comment:
I'm okay with using `dave/dst` here because we already use it so much in the rest of the project; but since we don't manipulate comments much, we *could* have decided to instead use the standard `go/ast`.

Reply:
OK, makes sense! let me look into changing that.